src/database/db.ts

```diff
@@ -4,21 +4,26 @@ import { TinyPg } from 'tinypg';
 import { getLogger } from '../logger';
 
 const initializationDirectory = path.join(__dirname, 'initialization');
+const logger = getLogger('db');
 
 const db = new TinyPg({
 	root_dir: [path.resolve(__dirname, 'queries')],
 });
 
+db.pool.on('connect', (client) => {
+	client.query("SET TIME ZONE 'UTC';").catch((err: unknown) => {
+		logger.error({ err }, 'Failed to set timezone');
```

Comment:
I think this falls in the category if "I don't feel like writing a test that will simulate a database failure" but if anybody feels strongly to the contrary I can do it (it has been done in this code base!)